src/checks/graphiteThreshold.check.js

```diff
@@ -53,7 +53,7 @@ class GraphiteThresholdCheck extends Check {
 					return result.datapoints.some(value => {
 						return this.direction === 'above' ?
 							value[0] && value[0] > this.threshold :
```

Comment:
Should we do it for this one as well? I don't think it technically needs it, but might make a bit more sense if they're both the same